src/shared/tpm2-util.c

```diff
@@ -1933,6 +1933,66 @@ static int tpm2_get_name(
         return 0;
 }
 
+/* Extend 'digest' with the PolicyAuthValue calculated hash. */
+int tpm2_calculate_policy_auth_value(TPM2B_DIGEST *digest) {
+        const TPM2_CC command = TPM2_CC_PolicyAuthValue;
```

Comment:
we usually don't use "const" on simple local variables. it doesn't really add safety.

but also doesn't matter...

Reply:
ack